nlp/2021_07_07_neural_keyword_extraction/neural_keyword_extraction.ipynb

```diff
@@ -0,0 +1,1520 @@
+        "\n",
+        "This makes `pke` a great choice to get started with keyword extraction, experiment with different methods and generate baselines to improve upon.\n",
+        "\n",
+        "But what if the required performance is not met by these *classical* methods?\n",
+        "\n",
+        "Newly-developed auspicious extraction methods fall into the category of **neural keyword extraction**.\n",
+        "\n",
+        "The methods often utilise **sequence-to-sequence models** based on recurrent neural networks (RNNs) or Long Short-Term Memory (LSTM). Their objective is to transform a sequence of input words (the given document) into an abstract intermediate representation and generate a sequence of keywords from it.\n",
+        "\n",
+        "These models do not use words or phrases directly, which enables them to generate unseen keywords as well. This is called **keyword generation** and combines abstractive as well as extractive keywords.\n",
+        "\n",
+        "They report **impressive performance** increases over the classical extraction methods.\n",
+        "\n",
+        "However, training such models\n",
+        "**requires a large collection of documents** and annotated keywords, since the final training step is usually supervised.\n",
+        "\n",
+        "In addition, many of the models' repositories are not well maintained which makes it **difficult to train** them, especially on different languages or domains. \n",
+        "\n",
+        "This leaves the quesition of how neural keyword extraction can already be used today.\n",
+        "\n",
+        "Follow the below sections in this notebook to learn how to use **two approaches in the neural keyword extraction category** and how they compare to classical extraction methods. 👇"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Ak0IG15fsUVV"
+      },
+      "source": [
+        "## 🏗 Getting started: Install packages & download models\n",
+        "\n",
+        "The below cells will setup everything that is required to get started with keyword extraction:\n",
+        "\n",
+        "* Install packages\n",
+        "* Download additional resources"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "qbMg6Xyj8Xww",
+        "outputId": "3b0b8599-77d5-41bf-b295-0ca49733b28e"
+      },
+      "source": [
+        "!pip install git+https://github.com/boudinfl/pke.git\n",
+        "!pip install transformers\n",
+        "!pip install keybert\n",
+        "\n",
+        "# Download additional resources\n",
+        "!python -m nltk.downloader stopwords\n",
+        "!python -m nltk.downloader universal_tagset\n",
+        "!python -m spacy download en # Download English model"
+      ],
+      "execution_count": 23,
+      "outputs": [
```

Comment:
Perhaps for cleanliness we can clear the output of this cell. 
It's also worth it to add a `-q` flag to the pip install statement to generally suppress output